src/renderer/features/servers/components/add-server-form.tsx

```diff
@@ -33,11 +33,11 @@ export const AddServerForm = ({ onCancel }: AddServerFormProps) => {
     const form = useForm({
         initialValues: {
             legacyAuth: false,
-            name: '',
+            name: !isElectron() ? window.SERVER_NAME : process?.env.SERVER_NAME?? '' ,
             password: '',
             savePassword: false,
-            type: ServerType.JELLYFIN,
-            url: 'http://',
+            type: !isElectron() ? window.SERVER_TYPE?.toLowerCase() : process?.env.SERVER_TYPE?.toLowerCase() ?? ServerType.JELLYFIN,
```

Comment:
Rather than just accepting a string, should probably check is this string one of the valid types (currently just `ServerType.JELLYFIN` and `ServerType.NAVIDROME`). You can even define a function that takes a string and returns a valid server type (Navidrome/Jellyfin) if the string is valid, or defaults to Jellyfin, that way you don't need to type check
```typescript
const getServerType = (value?: string) => {
    switch (value?.toLowerCase()) {
        case ServerType.NAVIDROME:
            return ServerType.NAVIDROME;
        default:
            return ServerType.JELLYFIN;
    }
};
```
(an example, there are other ways). This prevents having to do `as ServerType` later